react/components/form/ImageUploader/MediaCenter.js

```diff
@@ -0,0 +1,180 @@
+import React, { Fragment } from 'react'
+import { Button, Modal, Tabs, Tab, Spinner } from 'vtex.styleguide'
+import ImageIcon from '../../../images/ImageIcon'
+import Dropzone from './Dropzone'
+import Gallery from './Gallery'
+
+class MediaCenter extends React.Component {
+  state = {
+    currentTab: 1,
+    isLoading: false,
+    error: false,
+  }
+
+  handleClearImage = () => {
+    this.props.onChange(null)
+  }
+
+  handleGallerySelect = image => {
+    this.props.onChange(image)
+    this.props.closeModal()
+  }
+
+  handleImageDrop = async (acceptedFiles, rejectedFiles) => {
```

Comment:
We need to add a file size verification to prevent files above 4MB, displaying an error message when it occurs.

Reply:
Already been done by the Dropzone component